### PR TITLE
[975] New to Ethereum? Blurb re-add

### LINKED ISSUE #975: [v.0.17.0] No 'New to Ethereum' message on the connection modal
Related to #924

Open the connection modal

**AR:** No 'New to Ethereum' message on the connection modal

**ER:**  message is displayed at the bottom of the modal: 
'New to Ethereum?  Learn more about wallets', where 'Learn more about wallets' link, navigates to https://ethereum.org/en/wallets/

### PR DESCRIPTION
**Pull request by W3stside**
closes #975 

New to Ethereum?
image

now is:
New to decentralised applications?
image

### CODE REVIEW

**Comment by W3stside** [comment]
probably needs a conditional chain check for the case of no Ethereum chains... BAH

File: src/custom/components/WalletModal/index.tsx [hunk 1]
```diff
@@ -10,6 +12,21 @@ export const GpModal = styled(Modal)`
...
+  width: 100%;
+  margin: 16px 0 0;
+  text-align: center;
+  font-size: smaller;
+  line-height: 1.5;
+`
+
+const NewToEthereum = () => (
+  <Blurb>
+    <div>New to decentralised applications? &nbsp;</div>{' '}
```
Comment (alfetopito) [inline src/custom/components/WalletModal/index.tsx:25]: Double space? `&nbsp;` and `{' '}`

**Review by alfetopito — APPROVED** [review]
Minor comment regarding the spaces, otherwise good to go.

File: src/custom/components/WalletModal/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/components/WalletModal/index.tsx:25]: was auto by lintyboi

but i think the second can be safely removed

Comment (W3stside) [inline src/custom/components/WalletModal/index.tsx:25]:
```suggestion
    <div>New to decentralised applications?</div>{' '}
```

**Comment by W3stside** [comment]
merging @biocom @anxolin please post-merge

File: src/custom/components/WalletModal/index.tsx (hunk 1 above)
Comment (alfetopito) [reply src/custom/components/WalletModal/index.tsx]: And yet, you removed the first :rofl:

**Pull request merged**